client/src/componets/routes/CreateRecipe.js

```diff
@@ -8,7 +8,11 @@ function RecipeCreate() {
   const navigate = useNavigate();
   const [item, setItem] = useState({
     title: '',
-    link: '',
+    Rating: '',
+    cook_time: '',
+    image: '',
```

Comment:
make sure these items that we are passing to our database match our DB schema from models/item.js in the same order